### PR TITLE
Feature/restricted range

### PR DESCRIPTION
**Pull request by DanielReid**
See https://github.com/angular-slider/angularjs-slider/pull/583#event-1708606978 (made new clean branch)

### CODE REVIEW

File: src/rzslider.js [hunk 1]
```diff
@@ -680,43 +683,36 @@
...
+                case 12:
                   this.ticks = jElem
                   break
               }
             },
             this
           )
-
-          // Initialize position cache properties
-          this.selBar.rzsp = 0
```
Comment (ValentinH) [inline src/rzslider.js]: Why did you remove this part?

**Comment by ValentinH** [comment]
ALso, I check the fiddle: https://jsfiddle.net/2hxm9cop/1/. Why is the restricted color not red?
image

**Comment by DanielReid** [comment]
Apologies, I messed up while squashing the old branch. Thanks for reviewing with attention!

**Comment by ValentinH** [comment]
Perfect! Releasing it right now at 6.6.0 :)

**Comment by ValentinH** [comment]
Done 🎉
